airbyte-integrations/connectors/destination-scaffold-destination-python/README.md

```diff
@@ -1,6 +1,6 @@
-# Scaffold Destination Python Destination
+# Scaffold-Destination-Python Destination
```

Comment:
This is supposed to be in title case, and should not have hyphens in them, as showed in `plop`'s [README](https://github.com/plopjs/plop#case-modifiers). Maybe this is changed in the latest `plop` release? But I don't see that in their changelog though. I think we need to try to fix this before merging.

Reply:
`titleCase` is imported from the `title-case` library:
https://github.com/plopjs/node-plop/blob/master/src/baked-in-helpers.js#L2

and this library has not been updated for a year:
https://www.npmjs.com/package/title-case

Really strange where the hyphen is coming from.

Reply:
@tuliren Yes that's indeed by the change in `plop`. The previously used plop version 2.7.6 used `node-plop` in version (0.26.3) (see https://github.com/plopjs/plop/blob/v2.7.6/package-lock.json#L4569-L4570), which itself used `title-case` in the version 2.1.1. `title-case@2.1.1` did actually remove hyphens in word (the names always had hyphens actually in them as passed in by plop), which you can see [here](https://codesandbox.io/s/title-case-2-1-1-8gnnvk).

The upgraded plop version uses [`node-plop` in 0.30.0](https://github.com/plopjs/plop/blob/v3.0.5/package-lock.json#L5851-L5852), which itself uses [`title-case` in 3.0.3](https://github.com/plopjs/node-plop/blob/v0.30.0/package-lock.json#L6480-L6481). `title-case@3.0.3` though does not remove hyphens anymore: https://codesandbox.io/s/title-case-3-0-3-1k9p54, thus we'll see the hyphens plot has in the name.

In this new version the same transform is now called `capitalCase`, which is unfortunately not enabled in plop by default. If we want the same casing we need to register it as a custom function and switch all templates to use `capitalCase` instead of `titleCase`. I wasn't sure if it's important for us how that name is exactly written in the READMEs, I am fine with either solution. The current one will change the casing, and therefore touch the scaffold output, with the other option I'll need to touch all templates, and the plop config to register the functions, but therefore all the scaffold output will stay the same. Let me know which one you prefer more, please.

Reply:
Cool. Thanks for digging into this. I do think it's important to not introduce the hyphens, because those titles will be everywhere and hard to change. It's actually quite easy to replace `titleCase` with `capitalCase`. PR here: https://github.com/airbytehq/airbyte/pull/10654

Reply:
@timroes, wait, actually I don't see the hyphen from `title-case@3.0.3` in https://codesandbox.io/s/title-case-3-0-3-1k9p54

<img width="1115" alt="Screen Shot 2022-02-25 at 01 53 26" src="https://user-images.githubusercontent.com/1933157/155694454-f1ec90c4-fc9a-4d79-bf98-64e6dc2a3b6a.png">

Are we seeing different output?

Reply:
Sorry I accedentally saved that with `capitalCase` already the example, since I wanted to double check the correct method name. Now it's back to using `titleCase` also for the second string :)

Reply:
Got it. Right, I can see the hyphen now. Thanks.